src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx

```diff
@@ -0,0 +1,989 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This component seems to contain lots of duplicate code from the analyze page. Lets remove duplicate code